superset/cli.py

```diff
@@ -268,7 +268,7 @@ def import_datasources(path, sync, recursive=False):
     '--datasource-file', '-f', default=None,
     help='Specify the the file to export to')
 @click.option(
-    '--print', '-p',
+    '--print-stdout', '-p',
```

Comment:
Why changing this? It seems to change the option for no good reason. If someone out there has written a shell script with `--print` it won't trigger the option anymore.

Reply:
The function export_datasources parameter is print_stdout, if you use print, it will cause error, like:
``` 
    return callback(*args, **kwargs)
  File "/Users/yongjie.zhao/.virtualenvs/incubator-superset-py3/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/yongjie.zhao/.virtualenvs/incubator-superset-py3/lib/python3.6/site-packages/flask/cli.py", line 257, in decorator
    return __ctx.invoke(f, *args, **kwargs)
  File "/Users/yongjie.zhao/.virtualenvs/incubator-superset-py3/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
TypeError: export_datasources() got an unexpected keyword argument 'print'
```
and click version is 6.7